libs/sdk-ui-dashboard/src/model/commandHandlers/filterContext/attributeFilter/removeAttributeFiltersHandler.ts

```diff
@@ -66,6 +67,10 @@ export function* removeAttributeFiltersHandler(
             }),
             // remove filter's display form metadata object
             filterContextActions.removeAttributeFilterDisplayForms(removedFilter.attributeFilter.displayForm),
+            // remove reflect dashboard attribute filter config
+            attributeFilterConfigsActions.removeAttributeFilterConfig(
+                removedFilter.attributeFilter.localIdentifier!,
```

Comment:
Can you please explain why we need `!` at the end of `localIdentifier`?

Reply:
I've defined the `removeAttributeFilterConfig` method to accept a string value, since the `localIdentifier` property is optional, meaning it can be undefined. The '!' operator here is used to help the ts-check know that we ensure the value is not undefined. 
BTW, We have two options to handle this:

- Maintain the current implementation.
- Redesign the `removeAttributeFilterConfig` method to accept both `string` and `undefined`.

Which option do you prefer?

Reply:
ok then.Thanks!